HHS/OPRE-OPS
# Ops 665 auth bp and refresh

## What changed

Fixed refresh token/API.
Restructured auth into separate module.

N.B. More auth work to follow.

## Issue

https://github.com/HHS/OPRE-OPS/issues/665

## How to test

Previously it was only possible to be logged in for 30 min.  Now you should be able to stay logged in for 12 hours.

## Definition of Done Checklist
